posthog/hogql/functions/mapping.py

```diff
@@ -533,6 +533,7 @@ class HogQLFunctionMeta:
 HOGQL_AGGREGATIONS: Dict[str, HogQLFunctionMeta] = {
     # Standard aggregate functions
     "count": HogQLFunctionMeta("count", 0, 1, aggregate=True),
+    "COUNT": HogQLFunctionMeta("count", 0, 1, aggregate=True),
```

Comment:
why do we need this? maybe better to replace this directly in the chatgpt output

Reply:
This is not directly linked to ChatGPT, but it does like to use uppercase variants of SQL functions just because they're very standard – for the same reason we should support them for regular SQL writing.
This instance of uppercased version support is definitely a kludge (and I think we do the same with the `now` function), because the actual ClickHouse rules are slightly different: select functions (primarily standard SQL ones) are marked as case insensitive ([list of cases in the ClickHouse repo](https://github.com/search?q=repo%3AClickHouse%2FClickHouse%20%22FunctionFactory%3A%3ACaseInsensitive%22&type=code)). We don't have to take the same approach at all, but I think ultimately there should be a similar `case_sensitive=False` setting on HogQL functions such as `count` or `length`